tests/album-create-test.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-undef */
+const { expect } = require("chai");
+const request = require("supertest");
+const db = require("../db/index");
+const app = require("../src/app");
+
+afterEach(async () => {
```

Comment:
The beforeEach and afterEach should be included in the body of the describe block. Also, you need to initialise the albums variable on line 27 by declaring it with `let albums;` at the top of the describe block so it can be assigned here